tests/examples/helm-local/index.ts

```diff
@@ -54,3 +54,39 @@ const nginx = new k8s.helm.v2.Chart("simple-nginx-local", {
 const frontendServiceSpec = pulumi.all([namespaceName, nginx]).apply(([nsName, nginx]) =>
     nginx.getResourceProperty("v1/Service", nsName, "simple-nginx-local-nginx", "spec"));
 export const frontendServiceIP = frontendServiceSpec.clusterIP;
+
+// Deploy a duplicate chart with a different resource prefix to verify that multiple instances of the Chart
+// can be managed in the same stack.
+new k8s.helm.v2.Chart("simple-nginx-local", {
+    // Represents chart `stable/nginx-lego@v0.3.1`.
+    path: "nginx-lego",
+    values: {
+        // Override for the Chart's `values.yml` file. Use `null` to zero out resource requests so it
+        // can be scheduled on our (wimpy) CI cluster. (Setting these values to `null` is the "normal"
+        // way to delete values.)
+        nginx: { resources: null },
+        default: { resources: null },
+        lego: { resources: null }
+    },
+    transformations: [
+        // Make every service private to the cluster, i.e., turn all services into ClusterIP instead of
+        // LoadBalancer.
+        (obj: any) => {
+            if (obj.kind == "Service" && obj.apiVersion == "v1") {
+                if (obj.spec && obj.spec.type && obj.spec.type == "LoadBalancer") {
+                    obj.spec.type = "ClusterIP";
+                }
+            }
+        },
+        // Put every resource in the created namespace.
+        (obj: any) => {
+            if (obj.metadata !== undefined) {
+                obj.metadata.namespace = namespaceName
+            } else {
+                obj.metadata = {namespace: namespaceName}
+            }
+        }
+    ],
+    resourcePrefix: "dup"
```

Comment:
nit: can we move this prop to the top of the def? i missed it at first as it seemed buried under the `transformations`

Reply:
also, given that this dupe chart is the same as the orig, can we wrap this in a func that takes a `resourcePrefix` to minimize the LoC?